package.json

```diff
@@ -26,45 +26,44 @@
     "clean-dist": "rimraf dist",
     "copy-files": "cpy src/index.html src/favicon.ico dist",
     "docker:run": "webpack-dev-server --inline --progress --port 8080",
+    "functests": "npm-run-all -p webdriver-start testserver-start protractor",
     "lint": "tslint ./src/*",
-    "postinstall": "typings install",
     "prestart": "npm run clean-and-copy",
     "prebuild": "npm run clean-and-copy",
     "prebuild:prod": "npm run clean-and-copy",
+    "protractor": "sleep 15 && ./node_modules/protractor/bin/protractor protractor.config.js",
     "setup": "npm install && npm run validate",
     "start": "webpack-dev-server --inline --progress --host 0.0.0.0 --port 8088",
     "test": "karma start",
-    "typings": "typings",
+    "testserver-start": "webpack-dev-server --inline --progress --host 0.0.0.0 --port 8088",
```

Comment:
There is inconsistency in naming npm scripts, e.g. watch:test and docker:run vs. clean-dist and testserver-start